star-wars.Rmd

```diff
@@ -5,16 +5,32 @@ output: github_document
 
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(knitr)
 ```
 
 ## Most accomplished
 
 Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the fewest number of starships):
```

Comment:
This sentence doesn't make sense, most accomplished != fewest number of ship piloted!

```suggestions
Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the highest number of starships):
```

Reply:
corrected rationale of sentence